featurebyte/feast/patch.py

```diff
@@ -112,7 +112,7 @@ class DataFrameWrapper(pd.DataFrame):
 
     def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
-        self._alias: Dict[str, str] = {}
+        self.attrs["_alias"] = {}
```

Comment:
Store the `_alias` attribute in `pd.DataFrame.attrs` to avoid triggering `UserWarning` (https://github.com/pandas-dev/pandas/issues/49649#issuecomment-1422731731).